packages/wonder-blocks-dropdown/components/multi-select-menu.md

```diff
@@ -0,0 +1,241 @@
+            } else {
+                selectedItems.splice(selectedItems.indexOf(change.value), 1);
+            }
+        });
+        this.setState({
+           selectedItems: selectedItems,
+        });
+    }
+
+    render() {
+        return <MultiSelectMenu
+            items={[
+                {
+                    label: "Red",
+                    value: "1",
+                },
+                {
+                    disabled: true,
+                    label: "Orange",
+                    value: "2",
+                },
+                {
+                    label: "Yellow",
+                    value: "3",
+                },
+                {
+                    label: "Green",
+                    value: "4",
+                },
+                {
+                    label: "Blue",
+                    value: "5",
+                },
+            ]}
+            disabled={this.props.disabled}
+            onChange={(changes) => this.handleChanges(changes)}
+            placeholder={"Choose some colors"}
+            selectedItems={this.state.selectedItems}
+            selectItemType={"colors"}
+            style={{
+                width: 170,
+                maxWidth: 170,
+            }}
+        />;
+    }
+}
+
+class ExampleWithShortcuts extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            selectedItems: ["1"],
+        };
+    }
+
+    handleChanges(changes) {
+        console.log(changes);
+        console.log("changes happened!");
+        const {selectedItems} = this.state;
+        changes.forEach((change) => {
+            if (change.selected) {
+                selectedItems.push(change.value);
+            } else {
+                selectedItems.splice(selectedItems.indexOf(change.value), 1);
+            }
+        });
+        this.setState({
+           selectedItems: selectedItems,
+        });
+    }
+
+    render() {
+        return <MultiSelectMenu
+            items={[
+                {
+                    label: "Red",
+                    value: "1",
+                },
+                {
+                    label: "Orange",
+                    value: "2",
+                },
+                {
+                    label: "Yellow",
+                    value: "3",
+                },
+                {
+                    label: "Green",
+                    value: "4",
+                },
+                {
+                    label: "Blue",
+                    value: "5",
+                },
+            ]}
+            displaySelectionShortcuts={true}
+            onChange={(changes) => this.handleChanges(changes)}
+            selectedItems={this.state.selectedItems}
+            selectItemType={"colors"}
+        />;
+    }
+}
+
+<View style={[styles.row]}>
+    <ExampleNoneSelected />
+    <View style={[styles.strutLike]} />
+    <ExampleWithShortcuts />
+    <View style={[styles.strutLike]} />
+    <ExampleNoneSelected disabled={true} />
```

Comment:
Can you move these into separate examples?  It will keep the code snippet shorter and easier for devs to see what's going on.

Reply:
Friendly ping. 🙂